tsrc/gitlab.py

```diff
@@ -25,6 +25,10 @@ def __str__(self):
         return "%s - %s" % (self.status_code, self.message)
```

Comment:
related to your commit message:

I like how you handle when there are more than 100 persons ! :P